<?xml version="1.0" ?>
<launch>
    <arg name="camera_name" default="realsense_r200" />

    <node name="realsense_camera_r200_node" pkg="acrv_realsense_ros" type="acrv_r200_ros" output="screen" respawn="false">
        <param name="serial_number" type="string" value="2521008008" />
        <!-- <param name="rotate_image_180" type="bool" value="false" /> -->
        <param name="camera_name" type="string" value="$(arg camera_name)" />
        <!-- <param name="rgb_hd_camera_info_path" type="string" value="/home/adamwtow/ros_ws/src/acrv_apc_2017/acrv_cameras/acrv_realsense_ros/cfg/$(arg camera_name)/rgb_hd_camera_info.yaml" />
        <param name="depth_camera_info_path" type="string" value="/home/adamwtow/ros_ws/src/acrv_apc_2017/acrv_cameras/acrv_realsense_ros/cfg/$(arg camera_name)/depth_camera_info.yaml" /> -->
    </node>

    <node pkg="tf" type="static_transform_publisher" name="cartesian_realsense_link_broadcaster_image_pipeline"
            args="0 0 0 0 0 0 /realsense_wrist_local_rgb_optical_frame /realsense_r200_depth_optical_frame 100" />
</launch>
